packages/core/lib/testing/TestRunner.js

```diff
@@ -39,11 +39,11 @@ class TestRunner {
     this.TEST_TIMEOUT = (options.mocha && options.mocha.timeout) || 300000;
   }
 
-  disableChecks() {
+  disableChecksOnEventDecoding() {
     this.disableChecks = true; //used by Solidity testing due to empty string problem on Solidity <0.7.6
```

Comment:
sorry and maybe this is pedantic, but does it make sense to change these variable names too?

Reply:
right but looked at as a generic property of `TestRunner` (like `TestRunner.disableChecks`) it is not clear what it means

Reply:
I mean, that's true, but when is that situation going to come up?  Everywhere it appears in the code, it's made clear what it means.

Reply:
Ok, if you are convinced that it doesn't really matter then I won't hold it back.

Reply:
I mean the field can be renamed in the future, it's a purely internal thing, it's not like that would break anything.

Reply:
sounds good to me!